src/query/sql/src/planner/dataframe.rs

```diff
@@ -188,19 +169,55 @@ impl Dataframe {
         Ok(self)
     }
 
+    pub async fn count(mut self) -> Result<Self> {
+        let select_list = [SelectTarget::AliasedExpr {
+            expr: Box::new(Expr::FunctionCall {
+                span: None,
+                distinct: false,
+                name: Identifier::from_name("count"),
+                args: vec![],
+                params: vec![],
+                window: None,
+            }),
+            alias: None,
+        }];
+
+        let mut select_list = self
+            .binder
+            .normalize_select_list(&mut self.bind_context, &select_list)
+            .await?;
+
+        self.binder
+            .analyze_aggregate_select(&mut self.bind_context, &mut select_list)?;
+
+        let (scalar_items, projections) = self
+            .binder
+            .analyze_projection(&self.bind_context.aggregate_info, &select_list)?;
+
+        self.s_expr = self
+            .binder
+            .bind_aggregate(&mut self.bind_context, self.s_expr)
+            .await?;
+
+        self.s_expr = self.binder.bind_projection(
+            &mut self.bind_context,
+            &projections,
+            &scalar_items,
+            self.s_expr,
+        )?;
+        self.s_expr = self
+            .bind_context
+            .add_internal_column_into_expr(self.s_expr.clone());
+        Ok(self)
+    }
+
     pub async fn aggregate(
         mut self,
         groupby: Vec<Expr>,
-        select_list: Vec<Expr>,
+        columns: &[&str],
```

Comment:
How to use these method to aggregate `select c, sum(a + 3 ) from t group by c` ?

Reply:
Modify to this:

```rust
pub async fn aggregate(
        mut self,
        groupby: GroupBy,
        aggr_expr: Vec<Expr>,
        having: Option<Expr>,
    )
```